### PR TITLE
Features/fanta 9 backend architecture setup

### PR DESCRIPTION
**Pull request by suyangsong**
Hey guys, I started on the backend, the server right now just handles a POST to server/warehouse, and it only responds hi to you, it works on my end but if you guys want to test it go ahead.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
+app.use(cors());
+// respond with "hello world" when a GET request is made to the homepage
+app.get("/", function(req, res) {
+  res.send("hello world");
+});
+app.post("/warehouse", warehouseRoutes);
```
Comment (durph) [inline server.js]: should say app.use(.....)

File: routes/instock-routes.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+const express = require("express");
```
Comment (durph) [inline routes/instock-routes.js]: this file not necessary

File: server.js (hunk 1 above)
Comment (suyangsong) [reply server.js]: changed to app.use

File: routes/instock-routes.js (hunk 2 above)
Comment (suyangsong) [reply routes/instock-routes.js]: merged warehouseRoutes into in-stock routes

**Pull request merged**